stdlib/_stat.pyi

```diff
@@ -82,3 +82,22 @@ if sys.platform == "win32" and sys.version_info >= (3, 8):
     IO_REPARSE_TAG_SYMLINK: int
     IO_REPARSE_TAG_MOUNT_POINT: int
     IO_REPARSE_TAG_APPEXECLINK: int
+
+if sys.platform == "win32":
```

Comment:
This is not true, I can use these values on macos
<img width="752" alt="Снимок экрана 2021-12-31 в 0 34 27" src="https://user-images.githubusercontent.com/4660275/147789476-71cdd3cd-3621-4a99-a04e-7bb892641012.png">

Reply:
I merged because a) Akuli did this intentionally (see comments in stubtest allowlists), b) they're documented as Windows only (https://docs.python.org/3/library/stat.html), c) there genuinely doesn't seem to be a use for them on other platforms and maybe that could uncover a bug.

That said, typeshed tries to avoid editorialising too much and we accomplish that by sticking close to implementation details, so if you felt strongly we could make these definitions unconditional.

Reply:
Ok, thanks for the explanation! I don't know when and why these values are used, so I don't have any strong opinions here 🙂